cluster/gce/config-default.sh

```diff
@@ -326,28 +327,27 @@ fi
 #   new subnetwork will be created for the cluster.
 ENABLE_IP_ALIASES=${KUBE_GCE_ENABLE_IP_ALIASES:-false}
 NODE_IPAM_MODE=${KUBE_GCE_NODE_IPAM_MODE:-RangeAllocator}
-if [ ${ENABLE_IP_ALIASES} = true ]; then
+if [ "${ENABLE_IP_ALIASES}" = true ]; then
   # Number of Pods that can run on this node.
   MAX_PODS_PER_NODE=${MAX_PODS_PER_NODE:-110}
   # Size of ranges allocated to each node.
-  IP_ALIAS_SIZE="/$(get-alias-range-size ${MAX_PODS_PER_NODE})"
+  IP_ALIAS_SIZE="/$(get-alias-range-size "${MAX_PODS_PER_NODE}")"
+  export IP_ALIAS_SIZE
```

Comment:
nit: Why don't we add `export` to previous line like the other ones here?

Reply:
Thanks for the comment!

The reason is that shellcheck detects SC2155 if we adds `export` at the beginning of the line. 
```
In ./cluster/gce/config-default.sh line 333:
  export IP_ALIAS_SIZE="/$(get-alias-range-size "${MAX_PODS_PER_NODE}")"
         ^-----------^ SC2155: Declare and assign separately to avoid masking return values.
```

Reply:
Thanks for your explanation, I got it :-)